package.json

```diff
@@ -18,5 +18,9 @@
   "homepage": "https://github.com/USGS-VIZLAB/active-flood-viz#readme",
   "devDependencies": {
     "bower": "1.8.0"
+  },
+  "dependencies": {
+    "jsdom": "11.1.0",
```

Comment:
These are actually dev dependencies since they are only used in the build process.